src/sts.rs

```diff
@@ -0,0 +1,278 @@
+//! The AWS STS API.
+//!
+//! If you're using the service, you're probably looking for [StsClient](struct.StsClient.html).
+#![allow(unused_imports)]
+
+include!(concat!(env!("OUT_DIR"), "/sts.rs"));
+
+pub use self::credential::{
```

Comment:
The generated sts code gives us the StsClient and all the requests, responses and errors. As you'd expect.
What we don't get is all the glue to create providers that the rusoto-credential module can use with it's provider chain.

This file implemnts all that in a private module (self::credential), then rexports selected symbols. (That's just a defensive measure to avoid colliding with anything imported from the generated sts module.